$NetBSD: patch-src_modules_loaders_Makefile.am,v 1.1 2024/03/18 18:37:19 schmonz Exp $

Remove non-portable linker flag not used with most other modules.

--- src/modules/loaders/Makefile.am.orig	2024-03-18 18:26:17.872501876 +0000
+++ src/modules/loaders/Makefile.am
@@ -83,7 +83,7 @@ gif_la_LIBTOOLFLAGS  = --tag=disable-sta
 
 heif_la_SOURCES      = loader_heif.c
 heif_la_CPPFLAGS     = $(HEIF_CFLAGS) $(AM_CPPFLAGS)
-heif_la_LDFLAGS      = -module -avoid-version -Wl,-z,nodelete
+heif_la_LDFLAGS      = -module -avoid-version
 heif_la_LIBADD       = $(HEIF_LIBS) $(top_builddir)/src/lib/libImlib2.la
 heif_la_LIBTOOLFLAGS = --tag=disable-static
 
